### DISCUSSION TITLE
why o1-preview is unavailable model?

### DISCUSSION THREAD

**Discussion by bsbrother** [community/community#143594] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

1. My email got "[GitHub] You have been granted access to GitHub Models limited public beta"

2. When access https://github.com/marketplace/models/azure-openai/o1-preview/playground, chat error:
    bsbrother now
    hi
    o1-preview now
    Unavailable model: o1-preview

3. When run example code error:
     azure.core.exceptions.HttpResponseError: (unavailable_model) Unavailable model: o1-preview
     Code: unavailable_model
     Message: Unavailable model: o1-preview

thank you!

**Comment by KateCatlin** [comment]
Closing this post as a duplicate of this other post! https://github.com/orgs/community/discussions/143596
